<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyInvincibleLibrary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="insertar">
     <attribute name="title">
      <string>Insertar</string>
     </attribute>
     <widget class="QPlainTextEdit" name="insertar_nombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el nombre de la imagen</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="insertar_autor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el autor</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="insertar_ano">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el año de creación</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="insertar_descripcion">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese una descripción</string>
      </property>
     </widget>
     <widget class="QLabel" name="insertar_imagen">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Desliza la imagen para insertar aquí&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="insertar_boton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Insertar</string>
      </property>
     </widget>
     <widget class="QLabel" name="insertar_ruta">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>206</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ruta de la imagen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="insertar_galeria">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese la galería</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="visualizar">
     <attribute name="title">
      <string>Visualizar</string>
     </attribute>
     <widget class="QPushButton" name="visualizar_boton">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>350</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Visualizar</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="visualizar_galeria">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese la galería</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="visualizar_nombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el nombre de la imagen</string>
      </property>
     </widget>
     <widget class="QLabel" name="visualizar_ruta">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>290</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ruta de la imagen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="visualizar_imagen">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>94</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Imagen por visualizar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>651</width>
        <height>451</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modificar">
     <attribute name="title">
      <string>Modificar</string>
     </attribute>
     <widget class="QPlainTextEdit" name="modificar_descripcion">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese una descripción</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modificar_boton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="modificar_galeria">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese la galería</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="modificar_nombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el nombre de la imagen</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="modificar_autor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el autor</string>
      </property>
     </widget>
     <widget class="QLabel" name="modificar_ruta">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>206</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ruta de la imagen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="modificar_ano">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el año de creación</string>
      </property>
     </widget>
     <widget class="QLabel" name="modificar_imagen">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Imagen por modificar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="modificar_cargar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="eliminar">
     <attribute name="title">
      <string>Eliminar</string>
     </attribute>
     <widget class="QLabel" name="eliminar_ruta">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>206</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ruta de la imagen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="eliminar_imagen">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Imagen por eliminar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminar_cargar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="eliminar_galeria">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese la galería</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="eliminar_nombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el nombre de la imagen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminar_boton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
